iml-graphql-queries/src/lib.rs

```diff
@@ -3,6 +3,7 @@
 // license that can be found in the LICENSE file.
 
 pub mod log;
+pub mod server_profile;
```

Comment:
maybe, `server/profile.rs`, etc?

Reply:
Would make sense once there are more queries related to server.